FirebaseStorage/Sources/FIRStorageUtils.h

```diff
@@ -17,6 +17,7 @@
 #import <Foundation/Foundation.h>
 
 @class FIRStoragePath;
```

Comment:
Can this forward declaration now be removed?

Reply:
It's still used by other APIs in this file.